universal-application-tool-0.0.1/test/services/applicant/ApplicantDataTest.java

```diff
@@ -38,6 +38,21 @@ public void preferredLocale_defaultsToEnglish() {
     assertThat(data.preferredLocale()).isEqualTo(Locale.US);
   }
 
+  @Test
+  public void hasPath_withRepeatedEntity() {
+    ApplicantData data =
+        new ApplicantData(
+            "{\"applicant\":{\"children\":[{\"entity\":\"first child\", \"name\": { \"first\":"
+                + " \"first\", \"last\": \"last\"}}, {\"entity\": \"second"
```

Comment:
super nit (optional): Could we break this string so entity 0 is on one line, and entity 1 is on the next?

Reply:
done

Reply:
done